_episodes/01-jupyter-notebooks.md

```diff
@@ -209,6 +216,20 @@ Or use [named links][data_carpentry].
   </div>
 </div>
 
+## Getting HELP!!
+
+In a [Jupyter Notebook][jupyter], there is another option for the `help()`
```

Comment:
The phrase "there is another option..." feels slightly out of place here since the `help()` function has not actually been discussed.  I suggest adding a sentence introducing the `help()` function first, and splitting the sample output. So you would have:

1. Sentence introducing the `help` function
2. Sample output showing the result of calling `help(print)`
3. The existing text talking about notebook only use of `?`
4. Output showing the use of `?`
5. A callout drawing attention to the fact that using ? in a notebook creates a HTML formatted window rather than using the cell output panel.

Reply:
If we want to be pedantic, then all keyboard keys (mostly notebook commands) should be formatted following Carpentry style guidelines - using the `<kbd></kbd>` tags. See the [style guide episode](http://carpentries.github.io/lesson-example/06-style-guide/index.html) of the example lesson for details. However, in the sake of time I am happy to merge without these changes so long as we create an issue to update the formatting. It would make a good "first issue" candidate.